media/transcoder/api/Transcoder.Samples/CreateJobWithAnimatedOverlay.cs

```diff
@@ -75,14 +77,8 @@ public class CreateJobWithAnimatedOverlaySample
                     X = 0.5,
                     Y = 0.5
                 },
-                StartTimeOffset = new Google.Protobuf.WellKnownTypes.Duration
-                {
-                    Seconds = 5
-                },
-                EndTimeOffset = new Google.Protobuf.WellKnownTypes.Duration
-                {
-                    Seconds = 10
-                }
+                StartTimeOffset = Duration.FromTimeSpan(new TimeSpan(0, 0, 0, 5, 0)),
```

Comment:
Up to you whether you change this, but I think `TimeSpan.FromMilliseconds(5)` is clearer here (and elsewhere).

Reply:
Done